home/views.py

```diff
@@ -0,0 +1,9 @@
+
+
+# Create your views here.
+
+from django.views.generic import TemplateView
+
+
+class HomepageView(TemplateView):
+    template_name = "index.html"
```

Comment:
since your template is in the `templates/home` and the template path in settings.py should refer to the root template directory (`templates` instead of `templates/home`), this needs to be changed from `ìndex.html` to `home/index.html`, so the template is still found.

Reply:
done, as well as first one